{
    "type": "bundle",
    "id": "bundle--641df1d9-4a35-4a48-b222-f90963847328",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--cf2931b7-0860-4b67-81e6-f0735bffff4b",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:13:04.226998Z",
            "modified": "2021-03-13T20:13:04.226998Z",
            "name": "CVE-2018-13867",
            "description": "An issue was discovered in the HDF HDF5 1.8.20 library. There is an out of bounds read in the function H5F__accum_read in H5Faccum.c.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2018-13867"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}